app/models/post.rb

```diff
@@ -2,8 +2,12 @@ class Post < ApplicationRecord
   belongs_to :user, optional: true
   has_many :comments, foreign_key: 'post_id'
   has_many :likes, foreign_key: 'post_id'
+  validates :title, presence: true
+  validates :title, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] You have validated the inputs fields however, on the create post form, the validation is failing, please fix it.
 
![validate1](https://user-images.githubusercontent.com/55635977/147197840-e7bf7d69-2a6f-4ec7-920d-1cf1e2275017.png)

Reply:
Thank you, now this bug it's fixed too 😄